dagger/pipeline/tasks/batch_task.py

```diff
@@ -63,7 +65,10 @@ def __init__(self, name, pipeline_name, pipeline, job_config):
 
         self._executable = self.parse_attribute("executable")
         self._executable_prefix = self.parse_attribute("executable_prefix") or ""
-        job_name = "{}-{}".format(pipeline.name, self.parse_attribute("job_name"))
+        job_name_raw = f"""{pipeline.name.replace("-", "/")}/{self.parse_attribute("job_name")}"""
+        final_path = str(Path(job_name_raw).resolve())
```

Comment:
I'm nor sure resolve works here. We need relpath from the dags directory.
I think resolve will give you the full absolute path, right?
What we need is relative path from the dags directory.
So if job name is `../container` and the pipeline is in `/usr/local/airflow/dags/team/pipeline/pipeline.yaml` then job_name should be `team-container`

Reply:
You can get the dags dir from the config object